modules/data-loading/pages/data-streaming-connector/big-query.adoc

```diff
@@ -0,0 +1,269 @@
+= Querying and Streaming from Google BigQuery
+:description: Steps to query and streaming data from Google BigQuery using Data Streaming Connector.
+:sectnums:
+
+The Data Streaming Connector allows you to invoke SQL queries to your Google BigQuery dataset and stream the query results to TigerGraph’s internal Kafka server with a specified topic.
+You can then create and run a xref:kafka-loader/index.adoc[Kafka loading job] to load data from Kafka into your graphs.
+
+== Prerequisites
+* You should have one of the following authentication credentials:
+** Google Service Account credentials
+** Access and refresh Tokens
+
+== Procedure
+
+=== Specify connector configurations
+
+The connector configurations provide the following information:
+
+* Connector class
+* Connection URL
+* Value converter
+* SQL query statement
+* Connector properties
+
+==== Specify connector class
+The connector class indicates what type of connector the configuration file is used to create.
+Connector class is specified by the `connector.class` key.
+
+For streaming from Google BigQuery, the class is `io.confluent.connect.jdbc.JdbcSourceConnector`.
+
+[source,text]
+----
+connector.class=io.confluent.connect.jdbc.JdbcSourceConnector
+----
+
+==== Specify connection URL
+
+To stream data from BigQuery, Data Streaming Connector uses the https://cloud.google.com/bigquery/docs/reference/odbc-jdbc-drivers#current_jdbc_driver[BigQuery JDBC Driver].
+The connection URL requires the domain name or IP address of the BigQuery server, the port number, the ID of your BigQuery project, and your credentials for authentication.
+You can also supply other optional properties to the connection that are supported by https://storage.googleapis.com/simba-bq-release/jdbc/Simba%20Google%20BigQuery%20JDBC%20Connector%20Install%20and%20Configuration%20Guide_1.3.0.1001.pdf[the BigQuery JDBC connector]:
+
+[source.wrap,text]
+----
+connection.url=jdbc:bigquery://<host_url>:<port_number>;<property1>=<value1>;<property2>=<value2>;...
+----
+
+
+|===
+|Parameter |Description
+
+|`host_url`
+|Domain name or IP address of the BigQuery server.
+Required.
```

Comment:
It's always `https://www.googleapis.com/bigquery/v2`, I'm not sure if there's other endpoint. But we should tell users this one.